### PR TITLE
Added samples for expand cluster and add osd for ceph clusters

### PR DESCRIPTION
**Pull request by shtripat**
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
@Tendrl/tendrl-core plz review

File: etc/add_osd_ceph_sample.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            "storage_disks": [{
+                "device": "/dev/vdf",
+                "journal": "/dev/vdg"
+            }
+            ]
+        },
+    },
+    "TendrlContext.integration_id": "89604c6b-2eff-4221-96b4-e41319240240"
+},
+"node_ids": [],
```
Comment (r0h4n) [inline etc/add_osd_ceph_sample.py]: node_ids not required

File: etc/add_osd_ceph_sample.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+payload = {
+"run": "integrations.ceph.flows.AddOsds",
+"type": "node",
+"created_from": "API",
```
Comment (r0h4n) [inline etc/add_osd_ceph_sample.py]: remove created_from, only required for jobs coming from the API

Comment (shtripat) [inline etc/add_osd_ceph_sample.py]: done

File: etc/add_osd_ceph_sample.py (hunk 1 above)
Comment (shtripat) [inline etc/add_osd_ceph_sample.py]: will remove that

File: etc/add_osd_ceph_sample.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
+payload = {
+"run": "integrations.ceph.flows.AddOsds",
+"type": "node",
+"created_at": "2017-03-09T14:15:14Z",
+"username": "admin",
+"parameters": {
+    "TendrlContext.cluster_name": "MyCluster",
```
Comment (r0h4n) [inline etc/add_osd_ceph_sample.py]: TendrlContext.integration_id should be required

**Pull request merged**